Detector/DetCEPCv4/compact/SEcal05_siw_Barrel.xml

```diff
@@ -76,7 +88,8 @@
 
   <readouts>
     <readout name="EcalBarrelCollection">
-      <segmentation type="MegatileLayerGridXY"/>
+      <segmentation type="MegatileLayerGridXY" grid_size_x="10.1667*mm" grid_size_y="10.1667*mm"/>
+      <!--segmentation type="WaferGridXY" grid_size_x="10.1667*mm" grid_size_y="10.1667*mm"/-->
```

Comment:
Could you describe `10.1667*mm` in your pull request?

Reply:
@mirguest 

Yes, I have described. In fact, grid_size_x and grid_size_y are fake parameters here, and it will automatically calculated by `Ecal_cells_across_megatile`.  I have also update to fix errors on cell size. More detail can be found in description.